src/__tests__/components/TransactionListRow.test.js

```diff
@@ -0,0 +1,35 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { TransactionListRowComponent } from '../../components/themed/TransactionListRow.js'
+
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      cryptoAmount: 'string',
+      denominationSymbol: 'string',
+      fiatAmount: 'string',
+      fiatSymbol: 'string',
+      isSentTransaction: true,
+      requiredConfirmations: 11,
+      selectedCurrencyName: 'string',
+      thumbnailPath: 'string',
+      walletBlockHeight: 11,
+      // eslint-disable-next-line react/no-unused-prop-types
+      walletId: 'string',
+      // eslint-disable-next-line react/no-unused-prop-types
+      currencyCode: 'string',
+      transaction: 'string' | 11,
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.